recipes-openxt/openxt/openxt-keymanagement/key-functions

```diff
@@ -454,54 +446,88 @@ set_encrypted_key() {
 }
 
 is_xc_scheme() {
-    local root="${1:-/}"
+    local boot="${1:-/boot/system}"
     local part="${2:-/dev/xenclient/config}"
 
-    [ -e ${root}/boot/system/config/recovery-private-key.conf ] && {
+    [ -e ${boot}/config/recovery-private-key.conf ] && {
         cryptsetup luksDump ${part} | grep -q "Slot 7: ENABLED"
         return $?
     }
 
     return 1
 }
 
+recovery_unlock_key() {
+    local part="${1}"
+    local name="${2}"
+    local boot="${3}"
+    local key="${4}"
+    local extra="${5:-}"
```

Comment:
Does this mean "set extra to $5, unless $5 is empty, in which case set extra to empty"?
That wouldn't be any different than `local extra="${5}"` would it?